src/dmd/mars.d

```diff
@@ -2062,6 +2062,23 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             params.warnings = DiagnosticReporting.inform;
         else if (arg == "-O")   // https://dlang.org/dmd.html#switch-O
             params.optimize = true;
+        else if (startsWith(p + 1, "message-style="))
+        {
+            const style = p + 1 + "message-style=".length;
+
+            if (strncmp(style, "msvc", 4) == 0)
+            {
+                params.messageStyle = MessageStyle.msvc;
+            }
+            else if (strncmp(style, "gnu", 3) == 0)
+            {
+                params.messageStyle = MessageStyle.gnu;
+            }
+            else
+            {
+                error("unknown message style '%s'", style);
+            }
```

Comment:
don't really nee the braces for single statements.

Reply:
fixed